common/src/main/java/jp/co/soramitsu/common/domain/GetAppVersion.kt

```diff
@@ -0,0 +1,17 @@
+package jp.co.soramitsu.common.domain
+
+import jp.co.soramitsu.common.resources.ContextManager
+
+class GetAppVersion(private val contextManager: ContextManager) {
+    operator fun invoke() = contextManager.getContext().let {
+        val versionName = it.packageManager.getPackageInfo(it.packageName, 0).versionName
+        val regex = Regex("[^0-9]")
+        regex.replace(versionName, "")
+    }
+}
+
+fun isAppVersionSupported(minSupportedVersion: String?, appVersionName: String): Boolean {
+    val appVersion = appVersionName.replace(".", "").toInt()
```

Comment:
not a good idea 2.1.11 and 2.11.1 also 2.0.11 = 2011 is greater then 2.1.3 = 213 which is not in terms of app versions